hail/python/hail/docs/experimental/index.rst

```diff
@@ -5,6 +5,13 @@ This module serves two functions: as a staging area for extensions of Hail
 not ready for inclusion in the main package, and as a library of lightly reviewed
 community submissions.
 
+There are a few experimental submodules:
```

Comment:
I find it slightly confusing how the functions documented on this page relate to these submodules. Perhaps change this to something like "The experimental module contains several freestanding functions---documented below---as well as a few experimental submodules:"